packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -15,6 +15,12 @@ describe("Errors", () => {
 
             assert(testError.errorType === ContainerErrorType.genericError);
         });
+        it("Should preserve the stack", () => {
+            const originalError = new Error();
+            const testError = CreateContainerError(originalError);
+
+            assert(testError["stack"] === originalError.stack);
+        });
     });
 
     describe("DataProcessingError coercion", () => {
```

Comment:
```suggestion
    describe("DataProcessingError coercion", () => {
    it("Should preserve the stack", () => {
            const originalError = new Error();
            const testError = CreateProcessingError(originalError);

            assert(testError["stack"] === originalError.stack);
        });
```

For now -- I'll move `extractLogSafeErrorProperties` and `extractLogSafeMessageProperties` into a telemetry-utils and write tests there.